identifiers/country-cy/top_level.csv

```diff
@@ -1,2 +1,8 @@
 id,name
 ocd-division/country:cy,Cyprus
+ocd-division/country:cy/district:01,Lefkosia
+ocd-division/country:cy/district:02,Lemesos
+ocd-division/country:cy/district:03,Larnaka
+ocd-division/country:cy/district:04,Ammochostos
+ocd-division/country:cy/district:05,Pafos
+ocd-division/country:cy/district:06,Keryneia

```

Comment:
I suggest omitting the initial 0.

Reply:
Also, where do these numbers come from?

Reply:
These are the official ISO 3166-2:CY of these districts. Also with the leading 0. You can find the sources in the Readme.
https://en.wikipedia.org/wiki/ISO_3166-2:CY

Should we still omit the leading 0, although they are specifically listed as ISO 3166-2?

Reply:
All good - I missed the second link.